src/TesApi.Web/Controllers/TaskServiceApi.cs

```diff
@@ -210,13 +210,21 @@ public virtual IActionResult GetServiceInfo()
         {
             var serviceInfo = new TesServiceInfo
```

Comment:
Since the values in this object never change while the process is running, I'd suggest moving its creation to Startup.cs as a singleton and capture it in this object's constructor.